tests/src/test/java/alluxio/master/FileSystemMasterClientIntegrationTest.java

```diff
@@ -62,4 +66,26 @@ public void getFileInfoReturnsOnError() throws IOException, AlluxioException {
     fsMasterClient.getStatus(new AlluxioURI("/doesNotExist"));
     fsMasterClient.close();
   }
+
+  @Test(timeout = 300000)
```

Comment:
We have no control on the thread scheduling of the thread that runs the listStatus and the thread that start the master. I set the timeout to a large number to avoid flakiness. A smaller number should work. If you prefer, i can  change it